excel/beginner/intro-to-functions-and-formulas/what-are-formulas.md

```diff
@@ -0,0 +1,46 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# What are Formulas?
+
+---
+## Content
+
+Formulas are the building blocks of excel. They are used to calculate the values of a cell.
+
+Formulas can be simple or complex, depending on the user.

```

Comment:
Hmmm maybe we can remove this sentence as it's not particularly useful to learning about formulas (especially as the 2nd sentence)